modules/about-custom-installation.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installation/installing-customizations-cloud.adoc
+
+[id='about-custom-installation-{context}']
+= About the custom installation
+
+The custom installation is divided into four phases:
```

Comment:
We are calling these "installation targets" and they are called "config" (or maybe "install-config"), "manifests", "ignition-configs", and "cluster". Nouns make a little more sense here because that is what is being installed. We are working on those definitions in the [installer repo](https://github.com/openshift/installer/pull/100).